### PR TITLE
Edit security-getting-started-tutorial.adoc

### PR DESCRIPTION
**Pull request by rolfedh**
References https://issues.redhat.com/browse/QDOCS-564

### CODE REVIEW

File: docs/src/main/asciidoc/security-getting-started-tutorial.adoc [hunk 1]
```diff
@@ -4,39 +4,35 @@ and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 [id="security-getting-started-tutorial"]
-= Getting Started with Security using Basic authentication and Jakarta Persistence
+= Getting started with security using Basic authentication and Jakarta Persistence
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-getting-started-tutorial.adoc:7]: Can we keep `Security` ? It is a well known term in Quarkus, same as `Basic`, I believe we have a few `Security` words, `security` does not read as a dedicated area in Quarkus.  cc @rsvoboda

File: docs/src/main/asciidoc/security-getting-started-tutorial.adoc [hunk 2]
```diff
@@ -4,39 +4,35 @@ and pull requests should be submitted there:
...
 |===
 |Endpoint | Description
-|`/api/public`| The `/api/public` endpoint can be accessed anonymously.
-|`/api/admin`| The `/api/admin` endpoint is protected with role-based access control (RBAC).
-Only users granted with the `admin` role can access it.
-At this endpoint, the `@RolesAllowed` annotation enforces the access constraint declaratively.
-|`/api/users/me`| The `/api/users/me` endpoint is protected by RBAC.
-Only users that have the `user` role can access the endpoint.
-This endpoint returns the caller's username as a string.
+|`/api/public`| Accessible without authentication, this endpoint allows anonymous access.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-getting-started-tutorial.adoc:33]: This refactoring reads unusual and is looks complex, `access is regulated declaratively`, this is a bit too complex, IMHO the current text is simple and clear

File: docs/src/main/asciidoc/security-getting-started-tutorial.adoc [hunk 3]
```diff
@@ -596,15 +592,14 @@ user
...
-You have learned how to create and test a secure Quarkus application by combining the built-in xref:security-basic-authentication.adoc[Basic authentication] in Quarkus with the Jakarta Persistence identity provider.
+You have successfully learned how to create and test a secure Quarkus application. This was achieved by integrating the built-in xref:security-basic-authentication.adoc[Basic authentication] in Quarkus with the Jakarta Persistence identity provider.
 
 After completing this tutorial, you can explore more advanced security mechanisms in Quarkus.
 The following information shows you how to use `OpenID Connect` for secure single sign-on access to your Quarkus endpoints:
 
 * xref:security-oidc-bearer-token-authentication.adoc[OIDC Bearer token authentication]
 * xref:security-oidc-code-flow-authentication.adoc[OIDC code flow mechanism for protecting web applications]
-
+security-getting-started-tutorial.adoc
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-getting-started-tutorial.adoc:602]: Can you please explain this addition ?

File: docs/src/main/asciidoc/security-getting-started-tutorial.adoc (hunk 1 above)
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-getting-started-tutorial.adoc:7]: Or may be it is OK, not sure, I appreciate the RH standards requiring the lower case for regular words, but it seems like it is not

File: docs/src/main/asciidoc/security-getting-started-tutorial.adoc (hunk 2 above)
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-getting-started-tutorial.adoc:33]: Or may be I need to read it a few times, reads better after a 2nd try :-).

File: docs/src/main/asciidoc/security-getting-started-tutorial.adoc [hunk 4]
```diff
@@ -4,39 +4,35 @@ and pull requests should be submitted there:
...
 |Endpoint | Description
-|`/api/public`| The `/api/public` endpoint can be accessed anonymously.
-|`/api/admin`| The `/api/admin` endpoint is protected with role-based access control (RBAC).
-Only users granted with the `admin` role can access it.
-At this endpoint, the `@RolesAllowed` annotation enforces the access constraint declaratively.
-|`/api/users/me`| The `/api/users/me` endpoint is protected by RBAC.
-Only users that have the `user` role can access the endpoint.
-This endpoint returns the caller's username as a string.
+|`/api/public`| Accessible without authentication, this endpoint allows anonymous access.
+|`/api/admin`| Secured with role-based access control (RBAC), this endpoint is accessible only to users with the `admin` role. Access is regulated declaratively using the `@RolesAllowed` annotation.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-getting-started-tutorial.adoc:34]:
```suggestion
|`/api/admin`| Secured with role-based access control (RBAC), this endpoint is accessible only to users with the `admin` role. Access is controlled declaratively using the `@RolesAllowed` annotation.
```

Minor suggestion

**Comment by sberyozkin** [comment]
Thanks @rolfedh LGTM, a couple of open questions but otherwise a nice update as always

File: docs/src/main/asciidoc/security-getting-started-tutorial.adoc (hunk 1 above)
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-getting-started-tutorial.adoc:7]: Looking around, `SomeProject Security` is typical. Rolfe, IMHO we should keep capital `S` in `Security`. May be `Getting started with Quarkus Security ....` but we probably checked it with Rostislav, it did not show too well, I may be the guide name proved too long etc.

Comment (rolfedh) [reply docs/src/main/asciidoc/security-getting-started-tutorial.adoc:7]: Thanks. Updated. This is referring to Quarkus Security.

File: docs/src/main/asciidoc/security-getting-started-tutorial.adoc (hunk 3 above)
Comment (rolfedh) [reply docs/src/main/asciidoc/security-getting-started-tutorial.adoc:602]: Whoops! Accidental copy/paste. Removed.

File: docs/src/main/asciidoc/security-getting-started-tutorial.adoc (hunk 4 above)
Comment (rolfedh) [reply docs/src/main/asciidoc/security-getting-started-tutorial.adoc:34]: Thanks. Updated.

**Review by sberyozkin — APPROVED** [review]

**Comment by sberyozkin** [comment]
@rolfedh Are you awaiting for more reviews or is it ready to be merged ?

**Comment by MichalMaler** [comment]
@sberyozkin Hello there! I will review it soon and that we are good to go!
Will ping ya when ready :)

**Review by MichalMaler — APPROVED** [review]

**Comment by MichalMaler** [comment]
@rolfedh @sberyozkin My review is done. Looks good!

**Comment by sberyozkin** [comment]
Thanks @rolfedh @MichalMaler, as it happens, in meantime, the PR I created earlier in response to the QE doc review has been merged, so Rolfe, please resolve the conflict

**Comment by rolfedh** [comment]
Thank you for your review, @MichalMaler!
@sberyozkin I've resolved the conflict. Ready to merge.

**Pull request merged**